lib/send_sms.rb

```diff
@@ -0,0 +1,18 @@
+# Download the twilio-ruby library from twilio.com/docs/libraries/ruby
+require("bundler")
+Bundler.require()
+
+#'twilio-ruby'
```

Comment:
Should be able to just include 'Twilio-ruby' by adding a line to your gem file
`gem 'twilio-ruby', '~> 5.62.0'`